web/templates/message_body.hbs

```diff
@@ -31,6 +31,10 @@
     {{timestr}}
 </a>
 
+{{#if msg/locally_echoed}}
+    <span class="fa fa-circle-o-notch slow_send_spinner hidden"></span>
+{{/if}}
```

Comment:
Should this be inside the `message_controls` part of the message body?

Reply:
Or maybe if we're visibly placing this in the timestamp column, it'd make sense to put it inside the `message_time` container block?

Reply:
(I'm mostly worried about making it easy to avoid breaking this as we make tweaks to the message feed design).

Reply:
It makes sense implementation wise, but I am not sure how it would look, @lakshaykananiya can you post a screenshot for it?

Reply:
Hmm, putting it in `message_controls` will look something like this,
<img width="673" alt="image" src="https://user-images.githubusercontent.com/101464851/228626473-e8bac5a2-1530-490d-8097-ede3f560ed4a.png">

The decision here is to make weather we want the spinner on the extreme right (as currently in PR) or like this.

We cannot put it into the `message_time` container since its opacity is 0 till the message is successfully sent. Maybe we can define a new parent element that will contain both of them?

Reply:
It will look the same by the way if we decide to go with the second option.

Reply:
Since the spinner is an independent component, having it share a parent with `time` alone doesn't make sense since even though their placement is similar, using grid helps us place it where we want it independent of the order of the elements in HTML. So, using a parent div here would limit that ability.

Reply:
OK